### PR TITLE
fix(passcode): use interaction_jti instead of session_id

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Rename `sessionId` to `interactionJti` in passcode.

According to `node-oidc-provider`'s [doc](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md#user-flows), `session` is not create before signing in, so there is no `sessionId`. `interaction.jti` is what we need.

## Testing

pass previous UTs.

### CODE REVIEW

File: packages/core/src/lib/passcode.test.ts [hunk 1]
```diff
@@ -80,11 +80,11 @@ describe('createPasscode', () => {
 
   it('should disable existing passcode', async () => {
     const email = 'jony@example.com';
-    const sessionId = 'sessonId';
-    mockedFindUnconsumedPasscodesBySessionIdAndType.mockResolvedValue([
+    const jti = 'jti';
```
Comment (demonzoo) [inline packages/core/src/lib/passcode.test.ts:83]: What does jti stand for?

File: packages/schemas/tables/passcodes.sql [hunk 2]
```diff
@@ -2,7 +2,7 @@ create type passcode_type as enum ('SignIn', 'Register', 'ForgotPassword');
 
 create table passcodes (
   id varchar(128) not null,
-  session_id varchar(128) not null,
+  interaction_jti varchar(128) not null,
```
Comment (demonzoo) [inline packages/schemas/tables/passcodes.sql:5]: This is a postgres noob question: do we need to drop the table before create it with new columns?

**Review by demonzoo — APPROVED** [review]

File: packages/schemas/tables/passcodes.sql (hunk 2 above)
Comment (wangsijie) [reply packages/schemas/tables/passcodes.sql:5]: That is a question about migration, we have a linear project, that's hold it until v1 shipped, focus on code now.

File: packages/core/src/lib/passcode.test.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/lib/passcode.test.ts:83]: https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.7

Comment (demonzoo) [reply packages/core/src/lib/passcode.test.ts:83]: Got it, thanks!

File: packages/schemas/tables/passcodes.sql (hunk 2 above)
Comment (demonzoo) [reply packages/schemas/tables/passcodes.sql:5]: OK, got it.

**Pull request merged**